frontend/src/components/molecules/notification-with-buttons/AlertWithTwoButtons.js

```diff
@@ -14,7 +14,7 @@ export const AlertWithTwoButtons = ({
     <Alert type="warning" title={title}>
       {children}
     </Alert>
-    <div className="datapass-notification-buttons">
+    <div className="datapass-notification-buttons fr-btns-group--equisized">
```

Comment:
I am not found of mixing datapass classes with dsfr ones. This will break in future release of the dsfr.

plus, I do not understand what this add. Can you explain it to me ?